packages/python/libsql-client/tests/test_client.py

```diff
@@ -37,14 +37,6 @@ async def test_response_error(url):
             await client.execute("SELECT foo")
         assert "no such column: foo" in str(excinfo.value)
 
-@pytest.mark.asyncio
-async def test_http_invalid_url_error(http_url):
-    invalid_url = http_url + "/invalid/url"
-    async with libsql_client.Client(invalid_url) as client:
-        with pytest.raises(libsql_client.ClientHttpError) as excinfo:
-            await client.execute("SELECT 1")
-        assert "404" in str(excinfo.value)
```

Comment:
I assume that this test needed to be removed because we are now ignoring the path in the URL passed to the `Client`. Maybe we should not do this, and use relative paths in the `Client`?